content/en/docs/armory-admin/spinnaker-monitoring-and-metrics.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Spinnaker Monitoring and Metrics
```

Comment:
we may want to make this not-visible from the sidebar, not sure how to do that atm